backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphqlSchemaFieldFactory.java

```diff
@@ -667,9 +644,38 @@ private static void dropTables(
   }
 
   public GraphQLFieldDefinition.Builder schemaQuery(Schema schema) {
+    var builder =
```

Comment:
Please limit the use of var. Only use it if it is clear from the context what object is returned.

Reply:
so here it fine